### PR TITLE
add HashiCorp Nomad icon

### LINKED ISSUE #9518: Hashicorp Nomad
### Brand Name

Nomad

### Website

https://www.nomadproject.io/

### Popularity Metric

13.9k Stars on Github

### Official Resources for Icon and Color

https://www.hashicorp.com/brand

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by thiagogquinto**
nomad

**Issue:** closes #9518 

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5761,6 +5761,12 @@
             "source": "https://www.hashicorp.com",
             "guidelines": "https://www.hashicorp.com/brand"
         },
+        {
+            "title": "HashiCorp Nomad",
+            "hex": "60DEA9",
+            "source": "https://www.hashicorp.com/products/nomad",
+            "guidelines": "https://www.hashicorp.com/brand"
```
Comment (adamrusted) [inline _data/simple-icons.json:5768]:
```suggestion
            "title": "Nomad",
            "hex": "00CA8E",
            "source": "https://www.hashicorp.com/brand",
            "guidelines": "https://www.hashicorp.com/brand"
```

This'll need to be re-ordered alphabetically in order to keep the linter happy.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @thiagogquinto! 🎉
At a glance, it appears you've stretched the logo in both directions, meaning it's now distorted.
image
Could you please re-work this so that it still maintains it's aspect ratio, but the longest side (height) is 24px? I've also suggested changes to the JSON, to bring in line with our other HashiCorp icons.

File: _data/simple-icons.json (hunk 1 above)
Comment (thiagogquinto) [reply _data/simple-icons.json:5768]: Thanks for the feedback, working on it.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8951,6 +8951,12 @@
             "hex": "005AFF",
             "source": "https://www.nokia.com"
         },
+        {
+            "title": "Nomad",
+            "hex": "00CA8E",
+            "source": "https://www.hashicorp.com/products/brand",
```
Comment (adamrusted) [inline _data/simple-icons.json:8957]:
```suggestion
            "source": "https://www.hashicorp.com/brand",
```

File: icons/nomad.svg
Comment (adamrusted) [inline icons/nomad.svg:1]: I'm still seeing [slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/7432b528-6be4-4d7a-ba14-e19d2d7fede8) around the sides of the icon. Might be worth regenerating from source, or upping the precision to 4 decimals on [SVGOMG](https://jakearchibald.github.io/svgomg/)

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/nomad.svg
Comment (PratyushJayachandran) [inline icons/nomad.svg:1]: @thiagogquinto I still see [some differences.](https://github.com/simple-icons/simple-icons/assets/35000807/4de46da6-6a16-4ea8-bd41-11a1ad3dde01)
I compared the contributed SVG with this file `Nomad_PrimaryLogo_Color_RGB.svg` from their brand assets ZIP file.
Can you please take another look and correct the SVG.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @thiagogquinto?

**Comment by thiagogquinto** [comment]
> Are you still working on this one, @thiagogquinto?

Sorry for not informing before, but I am not.

File: icons/nomad.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nomad</title><path d="M12 0 1.5 6v12L12 24l10.5-6V6L12 0zm4.679 13.196-2.798 1.598-3.379-1.819v3.824l-3.177 1.995v-7.995l2.52-1.527 3.498 1.824V7.191l3.345-1.983-.009 7.988z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/nomad.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nomad</title><path d="m12.004 0-10.4 6v12l10.392 6 10.4-6V6L12.004 0zm4.639 13.204-2.77 1.6-3.347-1.832v3.826l-3.144 1.995V10.8L9.88 9.272l3.462 1.823V7.191l3.301-1.984v7.997z"/></svg>
```
Source: Nomad_PrimaryLogo_Color_RGB

**Comment by PratyushJayachandran** [comment]
@simple-icons/icon-review I committed changes to fix the issues. Please review.

**Comment by service-paradis** [comment]
Repo is pretty active, [~14.2k](https://star-history.com/#hashicorp/nomad&Date) Github stars with a [not so bad curve](https://star-history.com/#hashicorp/nomad&Date), I think this one can be considered for inclusion.

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**